fxmanifest.lua

```diff
@@ -11,7 +11,7 @@ shared_scripts {
 
     'config.lua',
     '@qb-core/shared/locale.lua',
-    'locales/en.lua', -- Change to the language you want
+    'locales/*.lua',
```

Comment:
```suggestion
    'locales/en.lua',
    'locales/*.lua',
```
`locales/en.lua` is also needed here.

Reply:
that's useless tho we're already calling it 🤔

Reply:
The other locales use English as their fallback language, so we have to ensure that it loads first.